sharding/collation.go

```diff
@@ -45,3 +54,83 @@ func (c *Collation) AddTransaction(tx *types.Transaction) {
 	// TODO: Include blob serialization instead.
 	c.transactions = append(c.transactions, tx)
 }
+
+// CreateRawBlobs creates raw blobs from transactions.
+func (c *Collation) CreateRawBlobs() ([]*utils.RawBlob, error) {
```

Comment:
No pointer receiver, please. We can assume that collations were constructed with the body and the list of transactions. Both are immutable so if the client needs a serialized data of transactions then they simply ready collation.Data()

Reply:
Alright removed it , now its just a value receiver